libs/wingsdk/src/shared/sandbox.ts

```diff
@@ -47,19 +47,29 @@ export class Sandbox {
       );
     }
 
+    let debugShim = "";
+    if (inspectorUrl?.()) {
+      // If we're debugging, we need to make sure the debugger has enough time to attach
+      // to the child process. This gives enough time for the debugger load sourcemaps and set breakpoints.
+      // See https://github.com/microsoft/vscode-js-debug/issues/1510
+      debugShim =
+        "\n  await new Promise((resolve) => setTimeout(resolve, 25));";
```

Comment:
It's unfortunate that this may have to be the solution since it's arguably an upstream issue. When debugging I think user's are more accepting of some slowness so I may even increase this a little bit.